pkg/manifest/anaconda_installer.go

```diff
@@ -135,7 +135,7 @@ func (p *AnacondaInstaller) getBuildPackages(Distro) []string {
 	packages := p.anacondaBootPackageSet()
 	packages = append(packages,
 		"rpm",
-		"lorax-templates-generic",
+		"lorax-templates-rhel", // XXX maybe always use -rhel?
```

Comment:
So, `-rhel` exists in Fedora; it seems to work. What do we do? At this point in the code we can't really switch on outside inputs.

Reply:
It really needs to know if it's building for RHEL or not and switch which one it uses.

Reply:
This is now added as a property on the image.